Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,24 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+  depends_on "doxygen" => :build
+  depends_on "cmake" # can this be :build ?
+  depends_on "gcc" # can this be :build ?
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "docs"
+    system "make", "install"
+
+    # although #{prefix} is not a "standard" one, we don't want/need
```

Comment:
You could also install the bash completions https://github.com/openkim/kim-api/blob/master/completions/completion.bash.in

Reply:
These are installed as part of "make install"